spec/requests/sign_in_spec.rb

```diff
@@ -11,38 +13,30 @@
   end
 
   context "when the user is signed in" do
-    let(:user_email_address) { "user@education.gov.uk" }
-
     before do
-      mock_auth_with_user(user_email_address)
+      mock_successful_authenticaiton(user.email)
+      allow_any_instance_of(ProjectsController).to receive(:user_id).and_return(user.id)
```

Comment:
Agree that this doesn't feel particularly great, but I guess it does solve the problem. We just need to remember to do it every time, but I can't think of a tidy alternative. We could have another helper that we pass the class into which handles this setup, but at this stage that feels like DRY for the sake of it.

Reply:
Yep, I feel like this must be an indicator we can do a better job in the implementation, but I can't figure it out, I am hoping that as we will have to use this a lot, one of us will hit on a good solution pretty quickly!